src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);

```

Comment:
You have initialized at the class-level and `getRandomColor()` method always will return the same color, because you have the same index on each call. You should declare another variable at the class-level. Please, think about random :)